human.txt

```diff
@@ -37,6 +37,7 @@ muan
 nik849
 petgoldfish
 pipmonkey
+pmayur
```

Comment:
I think my pmayur.html which happens to be my html file is conflicting where I made some changes intentionally. If thats not the problem can you help me with resolving the conflict?